configure

```diff
@@ -188,7 +196,8 @@ sudo chmod +x /bin/rpi-wifi.sh
 # EOF
 # sudo systemctl daemon-reload
 # sudo systemctl enable rpi-wifi.service
-crontab -l | { cat; echo "@reboot /bin/rpi-wifi.sh"; } | crontab -
+# crontab -l | { cat; echo "@reboot /bin/rpi-wifi.sh"; } | crontab -
+sed -i 's:# By default this script does nothing.:# By default this script does nothing.\n\n/bin/rpi-wifi.sh:' /etc/rc.local
```

Comment:
Why not something like?
```suggestion
sudo bash -c 'cat >> /etc/rc.local' << EOF
/bin/rpi-wifi.sh
EOF 
```

Reply:
Because the script won't finish... look at the last line `exit 0`

```
#!/bin/sh -e
#
# rc.local
#
# This script is executed at the end of each multiuser runlevel.
# Make sure that the script will "exit 0" on success or any other
# value on error.
#
# In order to enable or disable this script just change the execution
# bits.
#
# By default this script does nothing.


##### restart network interfaces to ensure right startup order ... ###
ifdown --force wlan0 && sudo ifdown --force ap0 && sudo ifup ap0 && sudo ifup wlan0
sysctl -w net.ipv4.ip_forward=1
iptables -t nat -A POSTROUTING -s 192.168.10.0/24 ! -d 192.168.10.0/24 -j MASQUERADE
systemctl restart dnsmasq
iwconfig wlan0 power off
###


# Print the IP address
_IP=$(hostname -I) || true
if [ "$_IP" ]; then
  printf "My IP address is %s\n" "$_IP"
fi

exit 0

Reply:
Oh, yes, I see. Not a clean way, but I don't see a better solution.